connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/RestServer.java

```diff
@@ -461,4 +469,18 @@ public static String urlJoin(String base, String path) {
             return base + path;
     }
 
+    /**
+     * Register header filter to ServletContextHandler.
+     * @param context The serverlet context handler
+     */
+    protected void configureHttpResponsHeaderFilter(ServletContextHandler context) {
+        String headerConfig = config.getString(WorkerConfig.RESPONSE_HTTP_HEADERS_CONFIG);
+        log.debug("headerConfig : " + headerConfig);
+        String[] configs = StringUtil.csvSplit(headerConfig);
+        Arrays.stream(configs)
+                .forEach(WorkerConfig::validateHttpResponseHeaderConfig);
```

Comment:
Is there a reason we don't want to validate these properties up front when all of the other configuration validation is being performed, via `ConfigDef.Validator` on `response.http.headers.config`? If we do that, we don't need this line.